src/components/modal/examples/components/AsyncModal.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <cdr-button @click="handleOpen">Open async modal</cdr-button>
+    <cdr-modal
+      label="Cat facts!"
+      :opened="modalOpened"
+      aria-describedby="description"
```

Comment:
this seems to be a location that automating what the describedby ID would make sense